content/docs/latest/spire-helm-charts-hardened-about/installation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installation
+short: Installation
+description: How to install the SPIRE stack using the SPIRE Helm charts
+kind: spire-helm-charts-hardened-about
+weight: 100
+aliases:
+    - /docs/latest/helm-charts-hardened/installation
+---
+
+## Non Production Deployment
+
+To do a quick install suitable for testing in something like minikube:
+
+```
+helm upgrade --install --create-namespace -n spire-server spire-crds spire-crds \
```

Comment:
Suggest to user `spire` or `spire-system` for the namespace as it will hold more than just the server.

Reply:
Updated